client/src/push/fcm.js

```diff
@@ -43,6 +43,8 @@ class FCMClient {
 
     self.notificationListener = this.FCN.onNotification(
       (notification) => {
+        console.log(notification);
```

Comment:
dont commit debug code (unless this was intentional)

Reply:
intentional. Have no use for the data in this PR but its important to see the full data that FCN got